docs/contracts/hash-consensus.md

```diff
@@ -16,3 +16,428 @@ the slot returned from `IReportAsyncProcessor.getLastProcessingRefSlot`.
 that have `consensusVersion` argument of the `HashConsensus.submitReport` call holding a diff.
 value than the one returned from `IReportAsyncProcessor.getConsensusVersion()`
 at the moment of the `HashConsensus.submitReport` call.
+
+## View methods
+
+### getChainConfig()
+
+Returns the immutable chain parameters required to calculate epoch and slot given a timestamp.
```

Comment:
I'd suggest to have the explainer with a scheme or a figure about timing terms and calculations (left the similar comment for `AccountingOracle`)